public/modules/custom/grants_oma_asiointi/src/Plugin/Block/AsiointirooliBlock.php

```diff
@@ -89,7 +89,7 @@ public function build() {
       $companyName = $selectedCompany['name'];
     }
 
-    $switchRole = Link::createFromRoute($this->t('Switch role'), 'grants_mandate.mandateform', [],
+    $switchRole = Link::createFromRoute($this->t('Switch role', array(), array('context' => 'Asiointirooli block')), 'grants_mandate.mandateform', [],
```

Comment:
FILE: ...stom/grants_oma_asiointi/src/Plugin/Block/AsiointirooliBlock.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------
 92 | ERROR | [x] Short array syntax must be used to define arrays
 92 | ERROR | [x] Short array syntax must be used to define arrays
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------